packages/genesys-spark-components/src/components/beta/gux-rich-text-editor/gux-rich-text-editor-action/gux-rich-text-editor-action.tsx

```diff
@@ -0,0 +1,70 @@
+import { Component, Element, Prop, h } from '@stencil/core';
+import { trackComponent } from '@utils/tracking/usage';
+
+import { buildI18nForComponent, GetI18nValue } from 'i18n';
+import translationResources from './i18n/en.json';
+import { GuxRichTextEditorActionTypes } from './gux-rich-text-editor-action.types';
+import {
+  calculateDisabledState,
+  returnActionTypeIcon
+} from '../gux-rich-text-editor.service';
+
+@Component({
+  tag: 'gux-rich-text-editor-action',
+  styleUrl: 'gux-rich-text-editor-action.scss',
+  shadow: true
+})
+export class GuxRichTextEditorAction {
+  private i18n: GetI18nValue;
+
+  @Element()
+  root: HTMLElement;
+
+  @Prop()
+  action: GuxRichTextEditorActionTypes;
+
+  @Prop({ mutable: true })
+  disabled: boolean = false;
+
+  @Prop()
+  isActive: boolean = false;
+
+  async componentWillLoad(): Promise<void> {
+    trackComponent(this.root, { variant: this.action });
+    this.i18n = await buildI18nForComponent(this.root, translationResources);
+    this.disabled = calculateDisabledState(this.root);
+  }
+
+  private renderTooltip(): JSX.Element {
+    if (!this.disabled) {
+      return (
+        <gux-tooltip>
```

Comment:
any reason to prefer gux-tooltip over the beta version here?

Reply:
No preference really will use the beta version.